ironfish/src/wallet/account.ts

```diff
@@ -19,6 +19,8 @@ import { WalletDB } from './walletdb/walletdb'
 
 export const ACCOUNT_KEY_LENGTH = 32
 
+export type AccountImport = { name: string; spendingKey: string }
```

Comment:
I'm not sure if we have a standardized pattern for re-usable types, so I wanted to specifically call this out - it feels kind of strange here but not sure if there's a better place for it.

Reply:
well, I don't really think we need this type exactly but it's fine either way. When you use it in importFile we just cast the result if JSON parsing to this type, but in reality it should be unknown and fed through a YUP schema.